test/rules/prefer-ignore-reason.test.js

```diff
@@ -0,0 +1,44 @@
+const {RuleTester} = require('eslint');
+const rule = require('../../rules/prefer-ignore-reason');
+
+const ruleTester = new RuleTester();
+
+ruleTester.run('prefer-ignore-reason', rule, {
+	valid: [
+		'/* istanbul ignore next: lazy to test */',
+		'/* istanbul ignore if: e2e */',
+		'// istanbul ignore next: with comment',
```

Comment:
```suggestion
		'// istanbul ignore else: with comment',
```

Might as well switch this so we have coverage for all 4 types of hints.